synfig-studio/brushes/CD_concept/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(BRUSH_CD_CONCEPT_FILES
+    04_pen.myb
+    04_pen_prev.png
+    08_pen.myb
+    08_pen_prev.png
+    Big_BG.myb
+    Big_BG_prev.png
+    Eraser.myb
+    Eraser_prev.png
+    readme.txt
+)
+
+set(CURRENT_OUTPUT_PATH "share/synfig/brushes/CD_concept")
+
+foreach (BRUSH_FILE IN ITEMS ${BRUSH_CD_CONCEPT_FILES})
```

Comment:
This code looks like a good candidate to make a function from it and use it everywhere to copy/install files.

Reply:
Also in this case why not just use a file(COPY...) to copy the entire directory?